data/publiccloud/terraform/azure.tf

```diff
@@ -202,7 +202,7 @@ resource "azurerm_linux_virtual_machine" "openqa-vm" {
     create = var.vm_create_timeout
   }
 
-  custom_data = var.cloud_init != "" ? filebase64(var.cloud_init) : null
+  #custom_data = var.cloud_init != "" ? filebase64(var.cloud_init) : null
```

Comment:
is it leftover of some debug or it is intended ? If intended why not just delete the line ?